apps/prairielearn/src/sprocs/sync_question_tags.sql

```diff
@@ -3,23 +3,19 @@ CREATE FUNCTION
         IN new_question_tags JSONB[]
     ) RETURNS void
 AS $$
-DECLARE
-    question_tags_item JSONB;
 BEGIN
-    FOREACH question_tags_item IN ARRAY new_question_tags LOOP
-        INSERT INTO question_tags (
-            question_id,
-            tag_id
-        ) SELECT
-            (question_tags_item->>0)::bigint,
-            tag_id::bigint
-        FROM JSONB_ARRAY_ELEMENTS_TEXT(question_tags_item->1) AS tag_id
-        ON CONFLICT (question_id, tag_id) DO NOTHING;
+    INSERT INTO question_tags (question_id, tag_id)
+    SELECT
+        (question_tags_item->>0)::bigint AS question_id,
+        JSONB_ARRAY_ELEMENTS_TEXT(question_tags_item->1)::bigint AS tag_id
```

Comment:
Neat! I never use this feature of a table generating function in a `SELECT`, but it does what we want. I should remember this.

Again, I think the `AS` clauses are irrelevant?

Reply:
Removed in e1e4645.